python/ray/tests/test_runtime_env_conda.py

```diff
@@ -55,6 +57,52 @@ def generate_runtime_env_dict(field, spec_format, tmp_path):
     return runtime_env
 
 
+@pytest.mark.skipif(
+    os.environ.get("CI") and sys.platform != "linux",
+    reason="Requires PR wheels built in CI, so only run on linux CI machines.")
+@pytest.mark.parametrize("field", ["conda", "pip"])
+def test_files_remote_cluster(start_cluster, field):
+    """Test that requirements files are parsed on the driver, not the cluster.
+
+    This is the desired behavior because the file paths only make sense on the
+    driver machine. The files do not exist on the remote cluster.
+    """
+    cluster, address = start_cluster
+
+    # cd into a temporary directory and pass in pip/conda requirements file
+    # using a relative path.  Since the cluster has already been started,
+    # the cwd of the processes on the cluster nodes will not be this
+    # temporary directory.  So if the nodes try to read the requirements file,
+    # this test should fail because the relative path won't make sense.
```

Comment:
did you verify this fails before the fix? not sure if we're making this an absolute path somewhere

Reply:
Yes, it fails without the fix